### PR TITLE
Expand the test to include other flags as well

### PR DESCRIPTION
**Pull request by wackxu**
**What this PR does / why we need it**:

Expand the test to include other flags as well

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```
 `NONE`

### CODE REVIEW

File: cmd/kube-apiserver/app/options/options_test.go [hunk 1]
```diff
@@ -28,13 +28,14 @@ import (
...
 	utilconfig "k8s.io/apiserver/pkg/util/flag"
+	restclient "k8s.io/client-go/rest"
 	kapi "k8s.io/kubernetes/pkg/api"
 	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
 	kubeletclient "k8s.io/kubernetes/pkg/kubelet/client"
 )
 
 func TestAddFlagsFlag(t *testing.T) {
-	// TODO: Expand the test to include other flags as well.
+
```
Comment (xiangpengzhao) [inline cmd/kube-apiserver/app/options/options_test.go]: remove the extra line please.

File: cmd/kube-apiserver/app/options/options_test.go [hunk 2]
```diff
@@ -116,16 +136,21 @@ func TestAddFlagsFlag(t *testing.T) {
 		EventTTL: 1 * time.Hour,
 		KubeletConfig: kubeletclient.KubeletClientConfig{
 			Port:         10250,
-			ReadOnlyPort: 10255,
+			ReadOnlyPort: 10256,
```
Comment (xiangpengzhao) [inline cmd/kube-apiserver/app/options/options_test.go]: why change this? it doesn't matter though.

File: cmd/kube-apiserver/app/options/options_test.go [hunk 3]
```diff
@@ -116,16 +136,21 @@ func TestAddFlagsFlag(t *testing.T) {
...
 				string(kapi.NodeHostName),
 				string(kapi.NodeInternalDNS),
 				string(kapi.NodeInternalIP),
 				string(kapi.NodeExternalDNS),
 				string(kapi.NodeExternalIP),
 			},
-			EnableHttps: true,
-			HTTPTimeout: time.Duration(5) * time.Second,
+			EnableHttps: false,
+			HTTPTimeout: time.Duration(6) * time.Second,
```
Comment (xiangpengzhao) [inline cmd/kube-apiserver/app/options/options_test.go]: and the two values here.

**Review by xiangpengzhao — COMMENTED** [review]
just nits:)

**Comment by wackxu** [comment]
@xiangpengzhao The parameter is the default value, I want to pass commands to verify those parameters, now I have to change back to the original value and fix the format

**Comment by xiangpengzhao** [comment]
Please run `./hack/update-bazel.sh` to update BUILD file(s)

**Comment by wackxu** [comment]
@xiangpengzhao Thank you and I have fix it.

**Comment by xiangpengzhao** [comment]
The code LGTM. But I will leave it to assignees to confirm and `/lgtm` :)

**Comment by ncdc** [comment]
LGTM. @liggitt ?

File: cmd/kube-apiserver/app/options/options_test.go [hunk 4]
```diff
@@ -28,13 +28,13 @@ import (
 	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 	utilconfig "k8s.io/apiserver/pkg/util/flag"
+	restclient "k8s.io/client-go/rest"
 	kapi "k8s.io/kubernetes/pkg/api"
 	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
 	kubeletclient "k8s.io/kubernetes/pkg/kubelet/client"
 )
 
 func TestAddFlagsFlag(t *testing.T) {
```
Comment (xiangpengzhao) [inline cmd/kube-apiserver/app/options/options_test.go]: another nit: it's strange the test name was `TestAddFlagsFlag`. I think `TestAddFlags` would be ok. WDYT?

Comment (wackxu) [inline cmd/kube-apiserver/app/options/options_test.go]: I agree with you. It looks like a typo and I will fix it.

**Comment by liggitt** [comment]
> LGTM. @liggitt ?

Didn't look closely, but seems ok

**Comment by wackxu** [comment]
@liggitt Please take a look at this PR, it needs your approve, thanks.

**Pull request merged**